### PR TITLE
[OSDEV-494] List upload: List uploads in the "PENDING" status are being displayed on the Contributor Profile page

### PR DESCRIPTION
**Pull request by roman-stolar**
[OSDEV-494](https://opensupplyhub.atlassian.net/browse/OSDEV-494) List upload: List uploads in the "PENDING" status are being displayed on the Contributor Profile page
- Filter facility list data that sent to User Profile by status equal APPROVED

### CODE REVIEW

File: src/django/api/serializers/user/user_profile_serializer.py [hunk 1]
```diff
@@ -72,6 +72,7 @@ def get_facility_lists(self, user):
                     source__contributor=contributor,
                     source__is_active=True,
                     source__is_public=True,
+                    status=FacilityList.APPROVED,
```
Comment (killev) [inline src/django/api/serializers/user/user_profile_serializer.py:75]: @roman-stolar , Frankly saying I doubt that only approved need to be shown. It's not clear from the task. Let's clarify with @mariel-oar

**Review by killev — CHANGES_REQUESTED** [review]
In general it looks good. but I'm not sure this is what really expected to be implemented in the original issue.

**Review by killev — APPROVED** [review]
LGTM.

**Review by VadimKovalenkoSNF — APPROVED** [review]
Approved

**Pull request merged**